axon-server/configuration.md

```diff
@@ -1,138 +1,424 @@
+      <td style="text-align:left">initial-nr-of-permits</td>
+      <td style="text-align:left">Number of messages that the server can initially send to a client</td>
+      <td
+      style="text-align:left">1000</td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">nr-of-new-permits</td>
+      <td style="text-align:left">Additional number of messages that the server can send to a client</td>
+      <td
+      style="text-align:left">500</td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">new-permits-threshold</td>
+      <td style="text-align:left">When a client reaches this threshold in remaining messages, it sends a
+        request with additional number of messages to receive</td>
+      <td style="text-align:left">500</td>
+    </tr>
+    <tr>
+      <td style="text-align:left"><em><b>Messaging (Between nodes of an Axon Server Cluster)</b></em>
+      </td>
+      <td style="text-align:left"></td>
+      <td style="text-align:left"></td>
+      <td style="text-align:left"></td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">commandFlowControl.initial-nr-of-permits</td>
+      <td style="text-align:left">Number of command messages that the master can initially send to a replica</td>
+      <td
+      style="text-align:left">10000</td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">commandFlowControl.nr-of-new-permits</td>
+      <td style="text-align:left">Additional number of command messages that the master can send to replica</td>
+      <td
+      style="text-align:left">5000</td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">commandFlowControl.new-permits-threshold</td>
+      <td style="text-align:left">When a replica reaches this threshold in remaining command messages, it
+        sends a request with this additional number of command messages to receive.</td>
+      <td
+      style="text-align:left">5000</td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">queryFlowControl.initial-nr-of-permits</td>
+      <td style="text-align:left">Number of query messages that the master can initially send to a replica</td>
+      <td
+      style="text-align:left">10000</td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">queryFlowControl.nr-of-new-permits</td>
+      <td style="text-align:left">Additional number of query messages that the master can send to replica</td>
+      <td
+      style="text-align:left">5000</td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">queryFlowControl.new-permits-threshold</td>
+      <td style="text-align:left">When a replica reaches this threshold in remaining command messages, it
```

Comment:
I believe that instead of `...query messages...`, `...command messages...`  should be used